wordstats/language_data/t2l/code/README.md

```diff
@@ -0,0 +1,62 @@
+### cleaning_words.py
```

Comment:
thanks for the clarification. 
i was suspecting that it might be a problem on my side...
i've cloned the project again, and now it's all lowercase. 
all is well!